src/test/java/gov/usgs/wma/statistics/logic/MonthlyStatisticsTest.java

```diff
@@ -337,9 +367,11 @@ public void testMostRecentProvistionalNONE_monathlyStats() {
 		fillMarchData(valueOrder);
 		List<Value> monthSamples = new ArrayList<Value>(valueOrder);
 		StatisticsCalculator.sortByDateOrder(monthSamples);
-//		stats.overallStats(monthSamples, valueOrder, MediationType.AboveDatum);
 
-		Map<String, String> stat = stats.monthlyStats(monthSamples).get("3"); //MediationType.AboveDatum
+		boolean isCalc = stats.monthlyStats(monthSamples);
+		assertTrue(isCalc);
+		
+		Map<String, String> stat = builder.build().getMonthly().get("3").percentiles; // MediationType.AboveDatum
 
 //		assertNotNull(stat);
```

Comment:
Delete comments or un-comment if it's useful for the test

Reply:
agreed